src/ClientData/include/ClientData/CaptureData.h

```diff
@@ -165,8 +165,7 @@ class CaptureData {
 
   void FilterBrokenCallstacks() { callstack_data_.UpdateCallstackTypeBasedOnMajorityStart(); }
 
-  void AddUniqueTracepointEventInfo(uint64_t key,
-                                    orbit_grpc_protos::TracepointInfo tracepoint_info) {
+  void AddUniqueTracepointEventInfo(uint64_t key, TracepointInfo tracepoint_info) {
```

Comment:
Let's take the chance to rename to `AddUniqueTracepointInfo` (even if the name was there before, this seems to be the right change to do it).

Reply:
Done.